Dockerfile

```diff
@@ -51,7 +47,7 @@ RUN groupadd --gid 1001 puppetdev \
   && useradd --uid 1001 --gid puppetdev --create-home puppetdev
 
 # Prep for non-root user
-RUN gem install bundler -v 2.4.22 \
+RUN gem install bundler -v 2.6.9 \
```

Comment:
Older versions of bundler have some weird bugs around the gem platform stuff. Just updating this to latest didn't fix the issue I was seeing with nokogiri, but it also didn't hurt anything, so figured I might as well update it.